### DISCUSSION TITLE
Unable to change taskbar color

### DISCUSSION THREAD

**Discussion by Hypporama** [valinet/ExplorerPatcher#2556] [Q&A]
Hello everyone
I have been using your ExplorerPatcher program for a while but when I update Windows 11 instead of having the Windows color in the taskbar it becomes black.
Even if I change the colors in the Windows settings, nothing changes.
I did a search in French since I'm French but I can't find it.
Do you have a solution to offer me?
THANKS

Translated with google translate

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

Please include any other software you have installed that modifies the UI of windows.

**Comment by Hypporama** [comment]
Good morning
Forgive me for the delay in my response, I did not receive a notification.
Windows version: 11 Home 22H2
The EP version: 22621.1555.55.1

Classic Start Menu 4.3.1

In ExplorerPatcher I just modified the combination of the taskbar windows and the Windows 7 style icons it seems to me.

**Reply by pyrates999** [reply]
If you have classic shell installed, uninstall it. That is no longer maintained.

The replacement is open shell.

Update EP as you are running an old release.

Please post windows version and build. You can find this by running winver.
Please post explorer patcher version after you update it.

Please include what settings you changed in explorer patcher properties application.

Please include any other software you have installed that modifies the UI of windows.

**Comment by Hypporama** [comment]
Bonjour
Je vais donc désinstaller Classic start menu pourvior si cela viens bien de la.
EP est a jour
Windows 11 version 22h2 build 22621.1992
![Capture d'écran A](https://github.com/valinet/ExplorerPatcher/assets/152990478/ec134119-5d84-4113-b5d2-ab142444f5a2)
![Capture d'écran B](https://github.com/valinet/ExplorerPatcher/assets/152990478/59ce7c31-7732-405e-99a8-16b2f54e5ff6)
![Capture d'écran C](https://github.com/valinet/ExplorerPatcher/assets/152990478/a0f3bca1-83f2-4394-943e-521480850938)
![Capture d'écran D](https://github.com/valinet/ExplorerPatcher/assets/152990478/c4a5a15e-1386-4bf8-be39-94bdedd90b81)
![Capture d'écran E](https://github.com/valinet/ExplorerPatcher/assets/152990478/341953f4-8ef9-4be1-9daf-dfb8987490b3)
![Capture d'écran F](https://github.com/valinet/ExplorerPatcher/assets/152990478/e81db6b7-6c25-4efb-adba-a1eb8968813e)
![Capture d'écran G](https://github.com/valinet/ExplorerPatcher/assets/152990478/c96a1c82-aa78-4c2f-baa9-d9fccd5e32fe)
![Capture d'écran H](https://github.com/valinet/ExplorerPatcher/assets/152990478/e7cb552e-d74a-4721-a4e4-12db1fb62651)

**Comment by Hypporama** [comment]
Apres avoir desinstallé classic start menu cela a résolut le problème merci a toi !!!!
J'ai donc une question supplémentaire.
Est il possible avec EP de le configurer pour faire apparaitre le menu démarrer de Windows 7 ?

**Reply by OrthodoxWindows** [reply]
Je pense que vous pouvez réinstaller Open-Shell (classic start menu). Votre problème était probablement lié à l'option "Customize taskbar", que vous avez peut-être activé par erreur. Normalement, si cette option est désactivé, Open-Shell ne touche pas à la barre des tâches.

**Reply by Hypporama** [reply]
Ou est cette option ?

**Reply by OrthodoxWindows** [reply]
Dans les paramètres du menu démarrer OpenShell, onglet Taskbar, option Customize taskbar.

![explorer_2weVMeO9BZ](https://github.com/valinet/ExplorerPatcher/assets/100134023/c8252f81-1b54-460d-b858-fef9f9d16f7b)

**Reply by Hypporama** [reply]
Enfaite l'option étant désactivé je l'ai réactivé et ma barre est revu de la couleur du reste donc MERCI infiniment!!!!

**Comment by pyrates999** [comment]
Your windows version is from July 2023: https://support.microsoft.com/en-au/topic/july-11-2023-kb5028185-os-build-22621-1992-605fa18f-bd49-41d8-80b1-245080e26c3d

Please install the current updates for windows.

You can have the windows 7 style start menu by installing open shell instead: https://github.com/Open-Shell/Open-Shell-Menu
